terminator-commons/src/main/java/io/github/algomaster99/terminator/commons/fingerprint/provenance/Jdk.java

```diff
@@ -0,0 +1,10 @@
+package io.github.algomaster99.terminator.commons.fingerprint.provenance;
+
+import io.github.algomaster99.terminator.commons.fingerprint.classfile.ClassFileAttributes;
+
+public record Jdk(ClassFileAttributes classFileAttributes) implements Provenance {
```

Comment:
The difference between this type and `io.github.algomaster99.terminator.commons.fingerprint.JdkClass`?

Reply:
The class was meant for the jsonl writing, but we dont do this so we dont need this class anymore.